docs/7.x/changelog.md

```diff
@@ -1110,6 +1110,27 @@ to learn how to gain insight into how the cluster status changes over time.
 
 ## 5.5 Releases
 
+### 5.5.49 LTS (June 25th, 2020)
+
+#### Improvements
+
+* Set priority classes on critical system pods ([#1692](https://github.com/gravitational/gravity/pull/1692), [planet#681](https://github.com/gravitational/planet/pull/681), [logging#65](https://github.com/gravitational/logging-app/pull/65), [monitoring#175](https://github.com/gravitational/monitoring-app/pull/175)).
+* Update `gravity status` to display unhealthy critical system pods ([#1702](https://github.com/gravitational/gravity/pull/1702), [planet#682](https://github.com/gravitational/planet/pull/682), [satellite#233](https://github.com/gravitational/satellite/pull/233)).
+* Display a more detailed reason for degraded status in `gravity status` ([#1707](https://github.com/gravitational/gravity/pull/1707)).
+* Add `--since` flag to `gravity report` command to allow filtering collected logs by time ([#1719](https://github.com/gravitational/gravity/pull/1719)).
```

Comment:
I wonder if we should indicate in here at all that the default is now whatever it was set to?